Makefile

```diff
@@ -20,6 +20,13 @@ docker:
 	docker build --tag sail:0.1 .
 	@echo 'for example: docker run --volume `PWD`:/data/ sail:0.1 --help'
 
+install-deps:
+	@opam install . --deps-only
```

Comment:
This seems fine. I wonder if it would be better to submit directly upstream, since it's not particular to this "JSON" fork.

Reply:
Makes sense, I will close this PR and split them up as needed. This might not be a needed target all together.